.travis.yml

```diff
@@ -1,30 +1,17 @@
 language: ruby
-os: osx
-env: OSX=10.11
-osx_image: xcode7.3
-rvm: system
+rvm: 2.0.0
+os: linux
```

Comment:
It can be done see https://github.com/Homebrew/homebrew-science/blob/master/.travis.yml for example.